src/main/java/moze_intel/projecte/emc/ComplexGraphMapper.java

```diff
@@ -55,7 +55,7 @@ public ComplexGraphMapper(IValueArithmetic arithmetic) {
 						debugFormat("Set value for %s to %d because %d/%d Conversions solved\n", something.toString(), minValue, getNoDependencyConversionCountFor(something), getConversionsFor(something).size());
 					}
 				}
-				System.out.format("lookat: %d solvable: %d\n", lookAt.size(), solvableThings.size());
+				System.out.format("lookat: %d solvable: %d%n", lookAt.size(), solvableThings.size());
```

Comment:
For documentation purposes I'll re-answer even though we already talked about it

That's just the C++ one that is unsafe, Java memory access is strictly enforced (unless you use sun.misc.Unsafe lol) :D